sensor_msgs/PointCloud2 pt_cld
string camera_frame
float64 container_slice_resolution
int32 num_pts_return
pouring_unknown_geom/ScanBoxMsg container_scan_box
bool plot_container_points
---
bool success
pouring_unknown_geom/ContainerEdgeProfile profile
pouring_unknown_geom/ContainerEdgeProfile generating_cyl_points
